deploy-board/deploy_board/webapp/helpers/placements_helper.py

```diff
@@ -36,3 +36,10 @@ def get_by_provider_and_cell_name(request, provider, cell_name):
 
 def get_by_id(request, placement_id):
     return rodimus_client.get("/placements/%s" % placement_id, request.teletraan_user_id.token)
+
+
+def get_simplified_by_ids(request, placement_str, provider, cell):
+    current_placement_ids = set(placement_str.split(','))
+    all_placements = get_by_provider_and_cell_name(request, provider, cell)
+    return [{k: placement[k] for k in placement if k in ['id', 'capacity', 'abstract_name', 'provider_name']}
```

Comment:
nit - might still prefer `.get(k)` for user facing in case a key is missing. it seems a key should never be missing though...

Reply:
Line 44 `for k in placement` so k must exist.
Line 45 `placement['id']``id` is a DB id so it must exist.